pom.xml

```diff
@@ -225,6 +223,11 @@
             <artifactId>jakarta.mail-api</artifactId>
             <version>${jakarta.mail-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.angus</groupId>
+            <artifactId>angus-mail</artifactId>
+            <version>1.0.0</version>
+        </dependency>
```

Comment:
Dependency ist weiterhin als dependency von com.sun.xml.ws:rt deklariert, allerdings im runtime scope und somit zur Compile-Zeit nicht verfügbar. Aus diesem Grund können die IMAPMessage Mocks in den Unit-Tests nicht mehr genutzt werden. Diese dependency daher wieder entfernen und die problematischen Mocks auf mock(Message.class) umbauen. Produktivcode funktioniert weiterhin wie gewünscht.

Reply:
dependency entfernt + imports in MailReceiverTest, ZepProjektzeitDetailsMailMapperTest und ZepMailToCommentServiceTest entfernt und IMAPMessage.class auf jakarta.mail.Message.class geändert